gramps/gen/db/dummydb.py

```diff
@@ -115,7 +119,7 @@ def wrapped(*args, **keywargs):
 
 class MetaClass(type):
     """
-    transform class by wrapping it with a diagnostic wrapper (if __debig__ is
+    transform class by wrapping it with a diagnostic wrapper (if __debug__ is
     not set
```

Comment:
```suggestion
    not set)
```
Close parenthesis in comment

Reply:
OK.  I added this by mistake when I fixed the conflict.

Reply:
Added.